pontoon/contributors/views.py

```diff
@@ -310,24 +310,30 @@ def settings(request):
     all_locales = list(Locale.objects.all())
     all_locales.insert(0, default_homepage_locale)
 
-    # Set custom homepage selector value
-    custom_homepage_locale = profile.custom_homepage
-    if custom_homepage_locale:
+    # Set default for custom homepage locale based on name
+    custom_homepage_locale_object = get_locale_or_redirect(profile.custom_homepage)
+    custom_homepage_locale_name = custom_homepage_locale_object.name
```

Comment:
Why `name` (not unique), not `code` (unique)?

Reply:
From what I understood from our conversation yesterday, the name of the local rarely would ever be edited (in comparison to the locale code) so I thought that setting the custom and default homepage using the locale name would be more resilient to changes in the locale code. However, if it is not unique then that will cause issues so I will change the logic to use the locale code.

Reply:
Sorry, sounds like a misunderstanding. Locale names are probably edited more often than locale codes.

When you address the comment [above](https://github.com/mozilla/pontoon/pull/3023#discussion_r1402386812), you might even be good without either.